css/fullcalendar.scss

```diff
@@ -156,18 +156,65 @@
 	}
 
 	// Reminder icon on events with alarms set
-	.fc-event-nc-alarms {
-		.icon-event-reminder {
+	&.fc-event-nc-alarms, &.fc-event-nc-attendees {
```

Comment:
The notification icon was just hidden because this wasn't just `&.fc-event-nc-alarms` :see_no_evil: